src/demangle.cc

```diff
@@ -340,7 +340,7 @@ static void MaybeAppendWithLength(State *state, const char * const str,
 // A convenient wrapper arount MaybeAppendWithLength().
 static bool MaybeAppend(State *state, const char * const str) {
   if (state->append) {
-    int length = StrLen(str);
+    auto length = StrLen(str);
```

Comment:
This introduces an unconditional dependency to C++11. However, glog is a C++98/03 library.